app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -159,4 +159,38 @@ public function processInvitationcall($observer)
         }
     }
 
+    /**
+     * @throws Exception
+     */
+    public function sendSyncUrl(Varien_Event_Observer $observer) {
+        if (
+            Mage::getStoreConfig('webwinkelconnect/product_review_invites/enabled')
+            && Mage::getStoreConfig('webwinkelconnect/general/enabled')
+        ) {
```

Comment:
Invert the condition and turn it into a guard clause to reduce nesting.

Reply:
You should also check that the shop ID and API key are set.

It's probably better to add a `isProductReviewInviteEnabled` method to the helper class that does all these checks:

* Plugin enabled
* Product review invites enabled
* ID and API key configured